coder-sdk/error.go

```diff
@@ -36,7 +36,7 @@ type APIErrorMsg struct {
 // an *APIError.
 func NewHTTPError(resp *http.Response) *HTTPError {
 	var buf bytes.Buffer
-	_, err := io.CopyN(&buf, resp.Body, 1<<20)
+	_, err := io.Copy(&buf, resp.Body)
```

Comment:
👍 nice simplification. I can't think of a case where the error body would be excessively large.